main.go

```diff
@@ -70,6 +70,10 @@ func main() {
 			Usage: "Path to librdkafka configuration file",
 			Value: "librdkafka.json",
 		},
+		cli.StringFlag{
+			Name:  "bootstrap-servers, b",
+			Usage: "Initial list of brokers",
```

Comment:
ultra-minor: I'd add an example here like 

```suggestion
			Usage: "Initial list of brokers (Example: 'kafka1.example.com:9092,kafkaN.example.com'",
```